### ISSUE TITLE
Character Sheets keep resetting

### ISSUE THREAD

**Issue by deirakos** [OtherwiseJunk/ParanoiaPerfectEditionSystem-foundryvtt#33]
after closing a character sheet they return to their default values (empty).
no modules are installed and it is a fresh unmodified world

**Comment by OtherwiseJunk** [comment]
This is a known issue; I'm hoping to have a 0.2.0 version out by this weekend that will fix that and make a few other improvements including assignable avatars, and including your hurt/flag status in considerations for rolls.

**Comment by OtherwiseJunk** [comment]
@deirakos this ended up being an easier fix than I anticipated.

https://foundryvtt.com/packages/paranoia

Version 0.2.0 should be available now, please let me know if you continue to see this issue after updating.

if I don't hear back by next week I'll close this out. Thanks!

**Comment by deirakos** [comment]
Thanks for the quick reply and fix.

it works for the top part and the first tab but third part: wellness, part fourth: role in society and back of the sheet: the naughty side still reset for me

**Comment by OtherwiseJunk** [comment]
Are you seeing the fifth tab that was dded, the Foundry Data one with the avatar image? I'm wondering if the 0.2.0 version just has the wrong files packaged

**Comment by OtherwiseJunk** [comment]
On the 0.2.0 version I have deployed locally I am seeing my changes reflected, but I haven't yet tested installing the package from the store;

![image](https://github.com/OtherwiseJunk/ParanoiaPerfectEditionSystem-foundryvtt/assets/4574230/e980b28d-15a2-45b4-a45c-964ccb92ca12)
![image](https://github.com/OtherwiseJunk/ParanoiaPerfectEditionSystem-foundryvtt/assets/4574230/0e2ad116-485d-45ce-9759-84207aed04a0)
![image](https://github.com/OtherwiseJunk/ParanoiaPerfectEditionSystem-foundryvtt/assets/4574230/e76600b4-0833-4941-919d-5297bb2781f1)

Everything I have on the sheet localy persists even when bringing down a game and bringing it back up though 🤔so maybe the 0.2.0 package has the wrong files when pulling from foundry itself.

**Comment by OtherwiseJunk** [comment]
Interesting. I'll take a look later this evening, I might not have the access to the manifest configured correctly

**Comment by OtherwiseJunk** [comment]
I had friend give it an install on a fresh install and it seemed to install and persist, so I think whenever you give this a go it should work

**Comment by deirakos** [comment]
yeah it works now. dunno if it just took foundry some time to use the correct manifest
